setup.md

```diff
@@ -107,11 +107,13 @@ sudo canasta skin enable Vector -i canastaId
 * For more info on finishing up your installation, go to the "After installation" section.
 
 ### After installation
-There's several things you can do to polish up your wiki so it's ready for use:
+Once your wiki is installed, there are many customizations you can do:
 
-* To add popular extensions quickly, visit the [Canasta extensions](#enabling-extensions) page to explore your choices.
-* Add a skin to your wiki by choosing a skin and add a `wfLoadSkin` call to `LocalSettings.php`. For instance, to install Timeless, add: `wfLoadSkin( 'Timeless' );` to `LocalSettings.php`.
-* All `.php` files in the `config/settings/` directory will be loaded as if their contents were in `LocalSettings.php`. In addition, if you want to remove the Canasta footer icon, you can remove the `config/settings/CanastaFooterIcon.php` file.
+* Canasta contains around 10 skins and 150 extensions; it is easy to install any of these by adding the appropriate calls to wfLoadSkin() or wfLoadExtension(). See [Enabling skins](#enabling-skins) and [Enabling extensions](#enabling-extensions) for more information.
+* MediaWiki (and its extensions and skins) can be configured in all sorts of ways, usually within `LocalSettings.php` or similar files. MediaWiki alone has over 1,000 configuration settings; see [here](https://www.mediawiki.org/wiki/Special:MyLanguage/Category:MediaWiki_configuration_settings) for one listing of them.
+* Note that, within Canasta, in addition to editing `LocalSettings.php`, you can also add configurations by adding settings files with any name - as long as they end in `.php` - to the `config/settings/` directory.
```

Comment:
My above comment means that, here, you would eliminate "in addition to editing LocalSettings.php"